students/matiasli/Lesson03/mailroom.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# a_namesoace = {'Name', 'Donations'}
+
+a_namespace = [ ['Bob', [40, 5, 25]], ['Alice', [100, 100, 100]], ['Zin', [30, 15, 50]], ['Chloe', [40, 13, 19]], ['James', [15]] ]
+
+
+# prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”)
+response = 0
+name_found = False
+
+while response != '3':
+    response = input("Select number: \n 1 - Send thank you \n 2 - Create report \n 3 - quit> ")
+    if response == '1':
```

Comment:
it is also best for these to redirect to feature functions just like in the example:
https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/mailroom_tutorial.html#controlling-main-program-flow